lib/common/exceptions/external/backend_service_exception.rb

```diff
@@ -65,25 +79,20 @@ def code
       end
 
       # REQUIRED - This is the http status code.
-      # unless you've specified that you want the status code to be something other
-      # then 400 explicitly it will default to 400. IT WILL NOT DEFAULT to whatever
-      # was provided by the backend service, because the backend service response
-      # might not always be relevant
       def status
-        i18n_data[:status].presence || 400
+        i18n_data[:status] || 400
```

Comment:
I'm in favor of preserving 500 errors as 502s as the most generic catch-all case. I feel like the backends we talk to most commonly have the error of returning 200s with error bodies, but they don't generally mix up 500s and 400s too badly.